pkg/security/ebpf/probes/all.go

```diff
@@ -158,7 +158,7 @@ type MapSpecEditorOpts struct {
 func AllMapSpecEditors(numCPU int, opts MapSpecEditorOpts) map[string]manager.MapSpecEditor {
 	editors := map[string]manager.MapSpecEditor{
 		"syscalls": {
-			MaxEntries: getMaxEntries(numCPU, 2048, 8192),
+			MaxEntries: 1024,
```

Comment:
Are we sure about this ? If we kept the old size, worst case it would be "the same" as today ?

Reply:
Back to a preview value